pyveg/src/data_analysis_utils.py

```diff
@@ -839,6 +839,80 @@ def resample_time_series(df, col_name="offset50", period="D"):
     return new_series
 
 
+def variance_moving_average_time_series(df, col_name="offset50", lenght=1):
+    """
+    Calculate a variance time series using a moving average
+
+    Parameters
+    ----------
+    df: DataFrame
+        Dataframe with date as index
+    col_name: string,
+        Identifying the column we will pull out
+    lenght: int
+        Lenght to be used in the moving average
+    Returns
+    -------
+    new_series: pandas Series with datetime index, and one column, one row per date
+    """
+    try:
+        series = df[col_name]
+    except:
+        ValueError('Error column name: '+col_name+' not found')
+
+    # just in case the index isn't already datetime type
+    series.index = pd.to_datetime(series.index)
+
+    variance = series.rolling(lenght).var()
+
+    variance.name = col_name+"_var"
+
+    return variance
+
+
+def ar1_moving_average_time_series(df, col_name="offset50", lenght=1):
```

Comment:
Also in this function, we should maybe fix "lenght" => "length" everywhere?

Reply:
Good catch! I'll fix these before merging.